webpack/assets/javascripts/react_app/components/charts/StatisticsChartsListStyles.js

```diff
@@ -0,0 +1,27 @@
+export default {
```

Comment:
I'm sorry, but I still disagree that this is a good idea. 
It might work for Facebook, I don't claim to understand their problem domain, and I'm not even sure how much this is even being used by them (looking at their source code does not seem to show much inline styling), but I believe this is not a fitting solution for our case, and I will elaborate:

> React inline styles is sass on steroids.

Inline styles is not sass on steroids. It completely breaks the whole concept of **Cascading** Style Sheets, since inline styles always take precedence over any other CSS rules (even `!important`!). 

> Although the final artifact is the same, implementing css in js differs from writing styles directly in html tags. Using composable javascript objects provides a powerful, easy to use and less error prone tool for managing components styles.

I would not want to use inline styles no matter whether they are defined by JS or hand written in HTML for that exact same reason. I agree this is less error-prone then hand-written inline styles, but IMHO this is much more complicated then using SCSS, increasing the learning curve and adding complexity to the react code. This is also a very confusing place for styling, as most developers would not expect to go to JS files when wanting to change styling (and going through potentially a chain of components just to understand where a style came from).

> It is possible to build React components that allow style overriding. Have a look at the MessageBox component.

That is only true for overriding inside React code. 
This does not allow setting a style that will be consistent through all the application, both for react components and other components. For example, the MessageBox components will have inconsistent styling with other message boxes used in the application that don't make use of the react component. 
This also makes overriding styling in a plugin (or in a package consumer, in the case of a separate package) extremely complex when compared to overriding CSS styles. Practically the only way to do so is to create another react component that will inherit from these components just to change some style. Even using JS to change inline styling will not work as the element will be redrawn by react.

> Dependencies are more explicit.

CSS dependencies can also be specified easily in JS - see https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/bundle_select2.js#L3 for example.